RobotCode/src/main/java/frc/robot/Commands/ClimberCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.Commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Subsystems.ClimberSubsystem;
+import frc.robot.constants.ClimberConstants;
+
+public class ClimberCommand extends Command{
+    public ClimberSubsystem climber;
+    public DoubleSupplier leftInput, rightInput;
+
+    public final double DEADBAND_RANGE = 0.1;
+    public void ExtendRetractArm(ClimberSubsystem c1, DoubleSupplier leftInput, DoubleSupplier rightInput) {
```

Comment:
This looks like a constructor and you do need one. Remove the return type and change the name to ClimberCommand to make it a constructor.